src/webview/static/base.css

```diff
@@ -475,3 +475,74 @@ blockquote {
   height: 32px;
   fill: hsla(0, 0%, 100%, 0.75);
 }
+
+/* Spoiler styling */
+.spoiler-block {
+  border: #808080 1px solid;
```

Comment:
We consistently use `hsl` or `hsla` in our CSS. Looks like this webapp code does the same thing, so there's no reason not to here.